.github/workflows/linux_nightlies.yml

```diff
@@ -1,8 +1,8 @@
-name: Nightly Linux Unit Tests
+name: Nightly unit tests, linux
 
 on:
   schedule:
-      - cron: '0 7 * * *'
+      - cron: '0 3 * * *'
```

Comment:
FYI the cron is in UTC so this is actually 11 pm EST. I think before it was set to 3 am EST.

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events

Reply:
Oh 💩  I'm sorry! I'll change it back